plugins/experimental/uri_signing/README.md

```diff
@@ -130,7 +130,7 @@ The following claims are understood:
   - `cdniuc`: Validated last, after key verificationD. **Only `regex` is supported!**
   - `cdniets`: If cdnistt is 1, this must be present and non-zero.
   - `cdnistt`: If present, must be 1.
-  - `cdnistd`: If present, must be 0.
+  - `cdnistd`: Renwal token cookies will have cdnistd path segments of the request in their path attribute.
```

Comment:
sorry to fixate on this.  `renewal`

Reply:
No worries! Thanks for reviewing! This should _really_ be resolved now.